test/step.test.js

```diff
@@ -0,0 +1,22 @@
+const assert = require('assert');
+const math = require('mathjs');
+
+const stepThroughExpression = require('../lib/simplifyExpression/stepThrough');
+
+function testStepsInExpression(exprStr, outputStr) {
+  it ('test the steps in simplifying expression', function () {
+    assert.equal((stepThroughExpression(math.parse(exprStr))[0]['changeType']), outputStr);
```

Comment:
why doesn't it work for complex examples? wanna chat more about this on gitter? 

here on github is fine too, whatever works for you